test/e2e/installplan_e2e_test.go

```diff
@@ -3191,6 +3371,14 @@ var _ = Describe("Install Plan", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		AfterEach(func() {
+			err := c.KubernetesInterface().CoreV1().Namespaces().Delete(context.TODO(), ns.GetName(), metav1.DeleteOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			err = crc.OperatorsV1alpha1().InstallPlans(ns.GetName()).Delete(context.TODO(), installPlanName, metav1.DeleteOptions{})
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Shouldn't the InstallPlan delete call come before the namespace deletion call? Wouldn't the server reject this event if the namespace is in a terminating phase?

Reply:
Oh it looks like this was just copy-pasted from below.

Reply:
Fixed